.gitignore

```diff
@@ -6,3 +6,4 @@
 /drivers/
 /.php-cs-fixer.cache
 /.phpunit.result.cache
+/.idea
```

Comment:
Can you remove this change? https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/